plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -97,7 +99,10 @@
     protected final QueryBuilder queryBuilder;
     protected final String identifierQuote;
     protected final Set<String> jdbcTypesMappedToVarchar;
-    private final IdentifierMapping identifierMapping;
+    protected final IdentifierMapping identifierMapping;
+
+    public static final ColumnMetadata TRINO_PAGE_SINK_ID_COLUMN =
+            new ColumnMetadata("trino_page_sink_id", BigintType.BIGINT);
```

Comment:
name here needs to be more tricky than that. Ideally you should check if the column like this do not already exist in table and pick up different one if that is the case (just add some number suffix)

Reply:
Fair enough. I'll check the column names where i'm creating the table, and add an incrementing integer suffix until there is no collision.

Reply:
just pushed code that makes this change